test/topology_experimental_raft/test_raft_service_levels.py

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+import pytest
+import time
+import asyncio
+import logging
+from test.pylib.util import unique_name, wait_for_cql_and_get_hosts
+from test.pylib.manager_client import ManagerClient
+from test.pylib.internal_types import ServerInfo
+from cassandra import ConsistencyLevel
+from cassandra.query import SimpleStatement
+
+
+logger = logging.getLogger(__name__)
+
+async def trigger_snapshot(manager: ManagerClient, servers: [ServerInfo]) -> None:
```

Comment:
We already have a `trigger_snapshot` function in `test_raft_snapshot_request.py`. I suggest extracting that function to `test/pylib/util.py` and then using it in the test so that copy-pasted (or, worse, slightly different versions of) `trigger_snapshot` doesn't proliferate in our tests.

Reply:
I think it's already extracted in raft auth PR, so I'll reuse it when I rebase on that PR (if it's not, I'll extract it then)